cli/cli_test.go

```diff
@@ -430,37 +431,13 @@ func TestGetMenderDaemonPID(t *testing.T) {
 	assert.Error(t, updateCheck(cmdKill, cmdPID))
 }
 
-// Minimal init
-func TestInitDaemon(t *testing.T) {
-	// create directory for storing deployments logs
-	tempDir, _ := ioutil.TempDir("", "logs")
-	defer os.RemoveAll(tempDir)
-	app.DeploymentLogger = app.NewDeploymentLogManager(tempDir)
-	bootstrap := false
-	dualRootfs := installer.NewDualRootfsDevice(nil, nil, installer.DualRootfsDeviceConfig{})
-	d, err := initDaemon(&conf.MenderConfig{}, dualRootfs,
-		&runOptionsType{dataStore: tempDir, bootstrapForce: bootstrap})
-	require.Nil(t, err)
-	assert.NotNil(t, d)
-	// Test with failing init daemon
-	set := flag.NewFlagSet("", 0)
-	command := &cli.Command{Name: "daemon"}
-	ctx := cli.NewContext(nil, set, nil)
-	ctx.Command = command
-	runOpts := runOptionsType{
-		logOptions: logOptionsType{logLevel: "info"},
-	}
-	assert.Error(t, runOpts.handleCLIOptions(ctx))
-}
```

Comment:
Why was this removed? Is it not appropriate anymore?

Reply:
This seemed to me to be tested in `daemon_test.go` also. So I saw no need for it  to be duplicated in `cli_test`.

Reply:
:+1: